pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -38,7 +39,7 @@ var genericPreflightChecks = [...]Check{
 	},
 	{
 		configKeySuffix:  "check-bundle-cached",
-		checkDescription: "Checking if CRC bundle is cached in '$HOME/.crc'",
+		checkDescription: "Checking if CRC bundle is installed in '$HOME/.crc'",
 		check:            checkBundleCached,
 		fixDescription:   "Unpacking bundle from the CRC binary",
 		fix:              fixBundleCached,
```

Comment:
Here we have `installed`, `Cached` and `unpacking` in just a few lines, can you use this commit as an opportunity to make this consistent?

Reply:
I put installed